### ISSUE TITLE
Two Blank Line - PEP8

### ISSUE THREAD

**Issue by HaroonKhanDotNet** [microsoft/vscode-pylint#234]
Type: <b>Bug</b>

1. Define a Python function/class.
2. Write a global statement immediately after the function/class definition without a blank line inbetween.
3. Pylint should shout with yellow squiggly about the missing two blank lines between the function/class and the global statement (but it is not doing it).

Sample code snippet:
```python
def pylance_function():
    'docstring'
    num = 5
    my_name = 'Haroon'
    print(my_name.islower(), num)
IS_REGISTERED = True
```
Extension version: 2022.6.0
VS Code version: Code 1.74.2 (e8a3071ea4344d9d48ef8a4df2c097372b0c5161, 2022-12-20T10:29:14.590Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD A12-9700P RADEON R7, 10 COMPUTE CORES 4C+6G (4 x 2495)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|6.96GB (1.12GB free)|
|Process Argv|--crash-reporter-id 30090ab0-e793-4989-bbc6-e5f3e1eb30dc|
|Screen Reader|yes|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
vscccc:30610679
pyindex848:30577860
nodejswelcome1:30587005
282f8724:30602487
gswce1:30612156
3d0df643:30613357
89544117:30613380
fim-prod:30623723

```

</details>

**Comment by brettcannon** [comment]
Do you get the warning when you run Pylint manually from the terminal? And can you include your logs from the Output panel for the Pylint extension's channel?

**Comment by HaroonKhanDotNet** [comment]
![vscode_problems_panel_pylint_snapshot](https://user-images.githubusercontent.com/83098130/210697326-3923cb9f-8c60-4a50-a51d-376f49a43401.png)

![win_cmd_pylint_snapshot](https://user-images.githubusercontent.com/83098130/210697327-a5e88302-a374-43b6-9daf-2b8b53c2d8b7.png)

**Comment by karthiknadig** [comment]
This should be reported on pylint itself (https://github.com/PyCQA/pylint). We only report what `pylint` tell us.
